qml/pages/ResultPage.qml

```diff
@@ -93,15 +93,15 @@ Page {
                 /* workaround to modify qml array is to make a copy of it,
                    modify the copy and assign the copy back to the original */
                 var new_parameters = search_parameters
-                new_parameters.jstime.setMinutes(new_parameters.jstime.getMinutes() + 15)
+                new_parameters.jstime.setMinutes(new_parameters.jstime.getMinutes() + Storage.getSetting("search_interval"))
```

Comment:
QML saves times in decimal "real" numbers instead of integers. This calculation has inconsistent results when adding decimal to integer minutes. This could be fixed with parseInt().